godotproject/Player.tscn

```diff
@@ -413,7 +413,6 @@ position = Vector2( 0.446458, -7.12662 )
 texture = ExtResource( 2 )
 vframes = 6
 hframes = 5
-frame = 3
```

Comment:
Not sure this was an intentional change. I think this should be left as-is?

Reply:
Ill revert this as well